oslat-server-start

```diff
@@ -66,14 +66,16 @@ echo "TEST_TILES: ${TEST_TILES}"
 
 CMD="taskset -c ${CPUS}"
 CMD+=" stress-ng"
+CMD+=" --taskset ${CPUS}"
```

Comment:
hmm, what is the different between the `taskset -c` and this?

Reply:
none, same thing